server/controllers/memory.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert');
+const { mysql } = require('../qcloud.js');
+const trainingTableName = "Training";
+
+async function get(ctx) {
+  var memeory = await mysql(trainingTableName).orderBy('UpdatedTime', 'desc').select('*').limit(10);
```

Comment:
Yes, setting a limit is good. In the future we can do pagination.

Reply:
Good suggestion